frontend/src/shared/modules/filters/config/itemLabelRenderer/boolean.label.renderer.ts

```diff
@@ -0,0 +1,6 @@
+import { BooleanFilterValue } from '@/shared/modules/filters/types/filterTypes/BooleanFilterConfig';
+
+export const booleanItemLabelRenderer = (property: string, { value }: BooleanFilterValue): string => {
+  console.log(value);
+  return `<b>${property}:</b> ${value ? 'True' : 'False'}`;
```

Comment:
Currently we most labels as: Property: is true. I think except for the ones that have multiple values